rentals.rb

```diff
@@ -0,0 +1,12 @@
+class Rentals
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] This should be together with the ```attr_accessor``` and not with the ```attr_reader```. Kindly make this change.